Samples/2.0/docker-aspnet-core/API/API.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <DockerComposeProjectPath>..\docker-compose.dcproj</DockerComposeProjectPath>
+    <ServerGarbageCollection>false</ServerGarbageCollection> <!-- https://blog.markvincze.com/troubleshooting-high-memory-usage-with-asp-net-core-on-kubernetes/ -->
```

Comment:
This is interesting... But we know that Orleans will run slower without that, so I don't know if we should set this value here. 

See https://dotnet.github.io/orleans/Documentation/clusters_and_clients/configuration_guide/configuring_.NET_garbage_collection.htm

Reply:
Thanks for the review!
Pods tended to crash overnight without this setting (due to Out Of Memory errors) which confused both clusters (Kubernetes and Orleans). This led to many Pods creating/killed etc.
I *think* this will be fixed in .NET Core 3 ([source](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-preview-3/)). Maybe we can leave this now and remove it when .NET Core 3 shipped.